app/views/markets/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2 class="forms">Edit a Market</h2>
+
+
+<%= form_for @market, method: :patch, url: market_path(@market) do |f| %>
```

Comment:
I think we were under the assumption patch was used for updating a form, whereas post would be more for new.